README.md

```diff
@@ -4,35 +4,45 @@ Alpha AMBER UI
 [![Build status](https://badge.buildkite.com/42748a5ed2c1c77c1dd974c497f0f39eaf39d782a289d19a8e.svg)](https://buildkite.com/csv-alpha/amber-ui)
 
 ## Prerequisites
+If you're going to run the project with Docker, you only need to install the following prerequisites:
+* [Docker Engine](https://docs.docker.com/get-docker/) 
+* [Docker Compose](https://docs.docker.com/compose/install/)
 
-You will need the following things properly installed on your computer.
-
-* [Git](https://git-scm.com/)
+Otherwise, you need the following prerequisites installed:
 * [Node.js](https://nodejs.org/) (with NPM, see `.node-version`, install preferably with [nvm](https://github.com/creationix/nvm#install-script))
 * [Yarn](https://yarnpkg.com/) (Alternative to NPM. Not strictly required, but uses version locking and is much faster)
-* [Ember CLI](https://www.ember-cli.com/): `yarn global add ember-cli` (or `npm install -g ember-cli`)
+* [Ember CLI](https://www.ember-cli.com/)*: `yarn global add ember-cli` (or `npm install -g ember-cli`)
 
-## Installation
+*: Installing the Ember CLI is required if you're not using Docker, but it's also useful with Docker if you want to make use of its features like Code Generators.
 
-* `git clone <repository-url>` this repository
-* change into the new directory
-* `yarn add` (or `npm install`)
+## Installation
+### With Docker
+* Build the project using `docker-compose -f docker-compose.development.yml build ui`. This will install the dependencies and set up the image. If dependencies are updated/added, you need to run this command again.
+
+Tip: to run commands in a container, you need to find the container's ID using the `docker ps` command first. Copy the ID of the container with image "amber_development_api".
```

Comment:
Do we need this here? Only relevant for API I think?

Reply:
The part about running commands in containers? I think that's useful information here as well for example for running the linter locally